async/.gitmodules

```diff
@@ -0,0 +1,11 @@
+[submodule "third_party/abseil-cpp"]
+	path = third_party/abseil-cpp
+	url = git@github.com/abseil/abseil-cpp.git
+
+[submodule "third_party/tensorflow"]
```

Comment:
Yeah, I'm pretty sketched out by adding these large submodules to this repository, even under the `async/` folder. Anyone that clones the repo and isn't careful with their git commands will pull in all that code. The https://github.com/iree-org/iree-samples/tree/main/cpp folder uses CMake FetchContent to avoid some of those drawbacks.